cou/utils/juju_utils.py

```diff
@@ -710,3 +729,67 @@ async def _wait_for_active_idle() -> None:
                 raise WaitForApplicationsTimeout(msg) from error
 
         await _wait_for_active_idle()
+
+    async def wait_for_active_idle(
+        self,
+        timeout: int,
+        idle_period: int = DEFAULT_MODEL_IDLE_PERIOD,
+        apps: Optional[list[str]] = None,
+        raise_on_blocked: bool = False,
+        raise_on_error: bool = True,
+    ) -> None:
+        """Wait for application(s) to reach active idle state.
+
+        If no applications are provided, this function will wait for all COU-related applications.
+
+        :param timeout: How long (in seconds) to wait for the bundle settles before raising an
+                        WaitForApplicationsTimeout.
+        :type timeout: int
+        :param idle_period: How long (in seconds) statuses of all apps need to be `idle`. This
+                            delay is used to ensure that any pending hooks have a chance to start
+                            to avoid false positives.
+        :type idle_period: int
+        :param apps: Applications to wait, defaults to None
+        :type apps: Optional[list[str]]
+        :param raise_on_blocked: If any unit or app going into "blocked" status immediately raises
+                                 WaitForApplicationsTimeout, defaults to False.
+        :type raise_on_blocked: bool
+        :param raise_on_error: If any unit or app going into "error" status immediately raises
+                                 WaitForApplicationsTimeout, defaults to True.
+        :type raise_on_error: bool
+        """
+        await self.wait_for_idle(
+            timeout=timeout,
+            status="active",
+            idle_period=idle_period,
+            apps=apps,
+            raise_on_blocked=raise_on_blocked,
+            raise_on_error=raise_on_error,
+        )
```

Comment:
I think this is too verbose to just call `wait_for_idle` with the default values. I didn't see a difference to call it directly

Reply:
Because `wait_for_active_idle` exists already, I shouldn't delete it on a feature PR.
I try not to refactor and add feature at the same time.